llvm/test/CodeGen/RISCV/rvv/vp-select.ll

```diff
@@ -0,0 +1,10 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=riscv64 -mattr=+v -verify-machineinstrs | FileCheck %s
```

Comment:
```suggestion
; RUN: llc -mtriple=riscv64 -mattr=+v < %s | FileCheck %s
```

Don't need to add -verify-machineinstrs in any random test

Reply:
I specifically added it for this test since it's dealing with RVV instructions. After we migrated the RISCVInsertVSETVLI pass to operate on LiveIntervals it really helps to have the machine verifier check that they're intact. It's already caught quite a few issues just on the existing RVV codegen tests alone.

Reply:
They'll get caught in expensive_checks builds. Unless this test is specifically trying to stress a verifier error, which it isn't, it's just wasting test time

Reply:
This is already the de-facto for the RVV in-tree tests:
```
$ git grep "\-verify\-machineinstrs" -- llvm/test/CodeGen/RISCV/rvv | wc -l
    2017
```

I get your point about slow test times but we rely very heavily on this for coverage on RISCVInsertVSETVLI. Should we create a separate issue to discuss this outside of this PR

Reply:
Right I think this is a long standing issue, and we have thousands of violations. It defeats the point of having a separate EXPENSIVE_CHECKS behavior

Reply:
> Right I think this is a long standing issue, and we have thousands of violations. It defeats the point of having a separate EXPENSIVE_CHECKS behavior

Do we have a RISCV expensive checks bot?

Reply:
Wouldn't the x86_64 expensive checks bot catch this? https://lab.llvm.org/buildbot/#/builders/16

Reply:
> Wouldn't the x86_64 expensive checks bot catch this? https://lab.llvm.org/buildbot/#/builders/16

I think it would. I just wasn't sure what targets it builds.

Reply:
Created https://github.com/llvm/llvm-project/issues/102092 to separate out this discussion